docs/index.rst

```diff
@@ -2110,7 +2120,7 @@ Constants
 .. _const-procfs_path:
 .. data:: PROCFS_PATH
 
-  The path of the /proc filesystem on Linux and Solaris (defaults to
+  The path of the /proc filesystem on Linux, Solaris and AIX (defaults to
   ``"/proc"``).
   You may want to re-set this constant right after importing psutil in case
   your /proc filesystem is mounted elsewhere or if you want to retrieve
```

Comment:
Please also add the new `AIX` constant in the doc and mark it as `.. versionadded:: 5.4.0`.

Reply:
The docs have a section called `.. _const-oses:`. Is it ok if I add `AIX` to this section with:

    .. versionchanged:: 5.4.0 added AIX

in order to keep this const in the same group as the rest?